### ISSUE TITLE
What's missing?

### ISSUE THREAD

**Issue by danielvarga** [dblN/stochastic_depth_keras#5]
Thanks a lot for doing this. I might misunderstand something, but the chart seems to say 15% eventual validation error on CIFAR-10. The original yueatsprograms Torch implementation has 5.23% validation error. I believe this difference is too large to be attributed to the different number of resnet blocks, or the lack of augmentation. Is there some missing functionality or known bug responsible for the discrepancy?

**Comment by dblN** [comment]
@danielvarga 
Thanks for pointing out. I think it's because of l2 weight decay and learning rate. They use learning rate for cifar-10 as below.
- 0 ~ 50% of max epochs => learning rate == 0.1
- 50 ~ 75% => 0.01
- 75 ~ 100% => 0.001

I'll implement this later tonight.
